Formula/ldid.rb

```diff
@@ -19,6 +19,8 @@ class Ldid < Formula
 
   def install
     inreplace "./make.sh", %r{^.*/Applications/Xcode-5.1.1.app.*}, ""
+    inreplace "make.sh", "-mmacosx-version-min=10.4", "-mmacosx-version-min=#{MacOS.version}"
```

Comment:
This should be using `inreplace do` with `gsub`, since we're touching the same file twice. Mildly depressing audit didn't catch this.

Reply:
(Also needs reporting upstream somewhere please 😇)

Reply:
The problem for this one is that neither the homepage `https://cydia.saurik.com/info/ldid/` nor the self-hosted gitweb `https://git.saurik.com/ldid.git` offers some places to make a comment / bug report. 😂

Reply:
`saurik AT saurik DOT com` is your friend 😉. He's been pretty quiet lately but nonetheless at least worth emailing & leaving a note in the formula about having done so & on what date.